pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -993,3 +1010,95 @@ section.quote {
     width: 100%;
   }
 }
+
+// Resources tabs section
+// Works with resources_tabs.haml partial
+// See code.org/teach for an example
```

Comment:
Right now this is on `code.org/teach-draft` until we redirect the current Teach page to the redesigned one on mid-July.